assets/src/components/TableAssignment.js

```diff
@@ -32,7 +32,7 @@ const tableStyle = theme => ({
     verticalAlign: 'middle'
   },
   tableResponsive: {
-    // width: '70%',
+    width: '70%',
```

Comment:
Changing the width not take up the same, for now. Will decide to condense or expand one formatting and hover on the bars implemented

Reply:
Sounds good